chapters/pointer/QN003.c

```diff
@@ -0,0 +1,26 @@
+/**
+ *  Write a program to print the elements of array using pointers.
+ */
+
+#include <stdio.h>
+
+int main()
+{
+  int a[50], i, n;
+  int *p = &a[0];
```

Comment:
```suggestion
  int *p = a;
```
Try this once. Check if it works.